packages/ember-testing/tests/helpers_test.js

```diff
@@ -179,83 +185,83 @@ moduleFor('ember-testing: Helper setup', class extends HelpersTestCase {
     );
   }
 
-  [`@test #setupForTesting attaches ajax listeners`](assert) {
-    let documentEvents = jQuery._data(document, 'events') || {};
-
-    assert.ok(
-      documentEvents['ajaxSend'] === undefined,
-      'there are no ajaxSend listers setup prior to calling injectTestHelpers'
-    );
-    assert.ok(
-      documentEvents['ajaxComplete'] === undefined,
-      'there are no ajaxComplete listers setup prior to calling injectTestHelpers'
-    );
-
-    setupForTesting();
-
-    documentEvents = jQuery._data(document, 'events');
-
-    assert.equal(
-      documentEvents['ajaxSend'].length, 1,
-      'calling injectTestHelpers registers an ajaxSend handler'
-    );
-    assert.equal(
-      documentEvents['ajaxComplete'].length, 1,
-      'calling injectTestHelpers registers an ajaxComplete handler'
-    );
-  }
-
-  [`@test #setupForTesting attaches ajax listeners only once`](assert) {
-    let documentEvents = jQuery._data(document, 'events') || {};
-
-    assert.ok(
-      documentEvents['ajaxSend'] === undefined,
-      'there are no ajaxSend listeners setup prior to calling injectTestHelpers'
-    );
-    assert.ok(
-      documentEvents['ajaxComplete'] === undefined,
-      'there are no ajaxComplete listeners setup prior to calling injectTestHelpers'
-    );
-
-    setupForTesting();
-    setupForTesting();
-
-    documentEvents = jQuery._data(document, 'events');
-
-    assert.equal(
-      documentEvents['ajaxSend'].length, 1,
-      'calling injectTestHelpers registers an ajaxSend handler'
-    );
-    assert.equal(
-      documentEvents['ajaxComplete'].length, 1,
-      'calling injectTestHelpers registers an ajaxComplete handler'
-    );
-  }
-
-  [`@test Ember.Application#injectTestHelpers calls callbacks registered with onInjectHelpers`](assert) {
-    let injected = 0;
-
-    Test.onInjectHelpers(() => {
-      injected++;
-    });
-
-    this.runTask(() => {
-      this.createApplication();
-      this.application.setupForTesting();
-    });
-
-    assert.equal(
-      injected, 0,
-      'onInjectHelpers are not called before injectTestHelpers'
-    );
-
-    this.application.injectTestHelpers();
-
-    assert.equal(
-      injected, 1,
-      'onInjectHelpers are called after injectTestHelpers'
-    );
-  }
+  // [`@test #setupForTesting attaches ajax listeners`](assert) {
```

Comment:
Can you bring these tests back, but put a short circuit in when jQuery is disabled?

something like:

```js
['@test asdfasdf'](assert) {
  if (jQueryDisabled) {
    assert.expect(0);
    return;
  }
}
```

Reply:
So this one might be a bit difficult to get at the event listeners w/ `documentEvents = jQuery._data(document, 'events');` if jquery is enabled since it is now using `document.addEventListener` in setup-for-testing.  The only thing I was thinking was to track the events with a data structure of our own.  Thoughts?